packages/lesswrong/lib/collections/electionCandidates/schema.ts

```diff
@@ -101,6 +102,44 @@ const schema: SchemaType<DbElectionCandidate> = {
     canRead: ["guests"],
     canCreate: ["sunshineRegiment", "admins"],
   },
+  /** Whether this is the main fundraiser (that will be distributed among the winning candidates), as opposed to being a particular candidate */
+  isElectionFundraiser: {
+    type: Boolean,
+    canRead: ["guests"],
+    canCreate: ["sunshineRegiment", "admins"],
+    canUpdate: ["sunshineRegiment", "admins"],
+    ...schemaDefaultValue(false),
+  },
```

Comment:
This is what I did initially, then I decided it would be confusing if we come back to do this again in future (i.e. if we did come back to it it would likely be next year, by which time we will have forgotten about this workaround)